tests/EdFi.Ods.Api.IntegrationTests/AuthorizationViewHelper.cs

```diff
@@ -0,0 +1,108 @@
+        {
+            (int source, int target) = sourceTargetTuple;
+
+            string sqlQuery = "";
+            if (source != 0)
+            {
+                sqlQuery= "SourceEducationOrganizationId =" + source + " AND ";
+            }
+   
+            sqlQuery += "TargetEducationOrganizationId =" + target + ";";
+
+            var sql = @$"
+                SELECT SourceEducationOrganizationId
+                FROM auth." + viewName + " WHERE " + sqlQuery;
+
+            using var command = connection.CreateCommand();
+            command.CommandText = sql;
+            using var reader = command.ExecuteReader();
+
+            var sourceList = new List<int>();
+            while (reader.Read())
+            {
+                sourceList.Add(reader.GetInt32(0));
+            }
+            return sourceList;
+        }
+
+        public static int GetStudentUSI(IDbConnection connection, string studentUniqueID)
+        {
+            var sql = @$"
+                SELECT StudentUSI FROM edfi.Student
+                WHERE StudentUniqueId = '"+ studentUniqueID +"';";
+
+            using var command = connection.CreateCommand();
+            command.CommandText = sql;
+            return Convert.ToInt32(command.ExecuteScalar());
+        }
+
+        public static void ShouldNotContainDuplicate(IDbConnection connection, string viewName)
+        {
+            var actualTuples = QueryAuthorizationByViewName(connection,viewName);
+            actualTuples.ShouldBeFalse();
+        }
+        public static void ShouldContainStudentInSchoolOrDistrict(IDbConnection connection, string viewName , params (int, int)[] expectedTuples)
+        {
+            foreach (var eachTuple in expectedTuples)
+            {
+                (int source, int target) = eachTuple;
+                var actualTuples = GetRecordsInAuthorizationView(connection, viewName, eachTuple);
+                actualTuples.All(p => p == source).ShouldBeTrue();
+            }
+        }
+
+        public static void ShouldContainStudentInBothSchool(IDbConnection connection, string viewName, params (int, int)[]  expectedTuples)
+        {
+            foreach (var eachTuple in expectedTuples)
+            {
+                (int source, int target) = eachTuple;
+                var actualTuples = GetRecordsInAuthorizationView(connection, viewName, eachTuple);
+                actualTuples.All(p => p == source).ShouldBeTrue();
+            }
+        }
+
+        public static void ShouldNotContainStudentInOtherSchoolOrDistrict(IDbConnection connection, string viewName, params (int, int)[] sourceTargetTuple)
+        {
+            foreach (var eachTuple in sourceTargetTuple)
+            {
+                (int source, int target) = eachTuple;
+                var actualTuples = GetRecordsInAuthorizationView(connection, viewName, eachTuple);
+                actualTuples.All(p => p == source).ShouldBeTrue();
+            }
+        }
```

Comment:
other than function and parameter names these have no other differences? Should these just be two function one for checking if it does contain the correct tuples and one to check if it doesn't?